### ISSUE TITLE
The top icon cannot be clicked in the icon menu on the left.

### ISSUE THREAD

**Issue by kovaccs** [resoai/TileBoard#385]
If there are too many icons on the left, those that fall in the line of the header cannot be clicked. Thans for the fix! 
![tileboard-issue](https://user-images.githubusercontent.com/19958469/86221154-d0b7b900-bb84-11ea-96d3-5dc2f4b64efa.jpg)

**Comment by resoai** [comment]
I'm afraid this is just one of inherited flaws with the header. Did you consider having menu icons at the bottom of the screen rather than on the left?

```
   /* menuPosition: LEFT (default) or BOTTOM */
   menuPosition: MENU_POSITIONS.BOTTOM,
```

**Comment by kovaccs** [comment]
Yes, I've tried this before, but because of that I would have to completely redesign some screens. The original idea would be better.
Thank you, your rapid answer!

**Comment by kovaccs** [comment]
If I move menü to bottom, can you suggest how I could place each "page" in the center of the screen?
For examle this one:
![tileboard-issue2](https://user-images.githubusercontent.com/19958469/86265909-b9002500-bbc4-11ea-98d6-9779f7a8ad80.jpg)

**Comment by resoai** [comment]
It would probably be wrong to actually align every page, but you can add left margin. You can either add following to custom.css:

```
.page-align {
  margin: 20px;
}
```

Or add this after definition of `CONFIG` in `config.js`:

```
var s = document.createElement('style');
document.head.appendChild(s);
s.sheet.insertRule('.page-align {margin: 20px}');
```

You would need to play around with `20px` to see which margin works best for your dashboard.

**Comment by kovaccs** [comment]
Ok!  I'm trying....

**Comment by kovaccs** [comment]
May I have an other question?
How can I apply different margin settins to every single pages?

**Comment by resoai** [comment]
I'm afraid it is a single element for all pages. You could play around setting position of the first tile to something greater than 0, but I would honestly recommend investing a bit more time and placing tiles on each page to maximise space use.

**Comment by kovaccs** [comment]
I really appreciate your time. I will take your advice and redesign the dashboard accordingly.

**Comment by rchl** [comment]
This should now be fixed (by #387).

**Comment by kovaccs** [comment]
Thanks for the notification, it works for me too.
